models/articles.model.js

```diff
@@ -57,3 +57,14 @@ exports.addComment = (article_id, comment) => {
       return rows[0];
     });
 };
+
+exports.addArticleVotes = (article_id, votes) => {
+  return db
+    .query(
+      "UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *",
+      [votes, article_id]
+    )
+    .then(({ rows }) => {
+      return rows;
```

Comment:
As the model is where the data manipulation happens, here might be a good place to take the returned article out of the array its returned in from the query 🔍